deploy/osd-storage/config.yaml

```diff
@@ -0,0 +1,11 @@
+deploymentMode: SelectorSyncSet
+selectorSyncSet: 
+  applyBehavior: "CreateOnly"
```

Comment:
This is so a dedicated-admin user can setup their own private encryption keys in the future.

Reply:
Thanks for explaining!